lib/Madara.lua

```diff
@@ -109,6 +109,25 @@ function defaults:getPassage(url)
 	return table.concat(map(GETDocument(self.expandURL(url)):select("div.text-left p"), text), "\n")
 end
 
+local function img_src(e)
+	local srcset = e:attr("data-srcset")
+
+	if srcset ~= "" then
+		-- get largest image
+		local max, max_url = 0, ""
+
+		for url, size in srcset:gmatch("(.-) (%d+)w") do
+			if tonumber(size) > max then
+				max = tonumber(size)
+				max_url = url:sub(tonumber(url:find("http"))) -- url might be ", http ..." at the second loop
```

Comment:
```suggestion
		for url, size in srcset:gmatch(",? *(.-) (%d+)w") do
			if tonumber(size) > max then
				max = tonumber(size)
				max_url = url:sub(tonumber(url:find("http")))
```
Change the pattern to avoid the issue outlined in the comment.